rbpf-cli/src/main.rs

```diff
@@ -153,10 +153,10 @@ native machine code before execting it in the virtual machine.",
     let mut account_refcells = Vec::new();
     let default_account = RefCell::new(AccountSharedData::default());
     let key = solana_sdk::pubkey::new_rand();
-    let mut mem = match matches.value_of("input").unwrap().parse::<usize>() {
+    let (mut mem, account_lenghts) = match matches.value_of("input").unwrap().parse::<usize>() {
```

Comment:
nit: typo here and below

Reply:
Looks like the type was already there and I just extended it :-)